edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/EdgeOperator.cs

```diff
@@ -48,6 +48,9 @@ public class EdgeOperator : IKubernetesOperator, IRuntimeInfoProvider
         readonly string serviceAccountName;
         readonly Uri workloadUri;
         readonly Uri managementUri;
+        readonly string persistentVolumeName; // Either PV or SC
+        readonly bool persistentVolumeNameIsSc;
+        readonly uint persistentVolumeClaimSizeMb;
```

Comment:
Given that both PV name and SC name are optional I wonder if using `Option` for representing this is more appropriate. For example, if neither has been specified, how are we to interpret `persistentVolumeNameIsSc`? How about having the following instead and get rid of the `bool`?
```csharp
readonly Option<string> pvName;
readonly Option<string> scName;
```

Reply:
Or if you like long names, then:
```csharp
readonly Option<string> persistentVolumeName;
readonly Option<string> storageClassName;
```

Reply:
Agreed,  Refactored this class to have 2 options,